src/interface/cli/mod.rs

```diff
@@ -0,0 +1,86 @@
+use clap::{App, Arg, SubCommand, crate_authors, crate_version};
+use crate::readers;
+use crate::writers;
+use crate::writers::dir_builder::DirectoryBuilder;

```

Comment:
(minor) 
could be condensed into -> `use crate::writers{self, dir_builders::DirectoryBuilder}`